Gemfile.lock

```diff
@@ -378,13 +379,13 @@ GEM
       nokogiri (~> 1.6.0)
       ruby_parser (~> 3.5)
     html_validation (1.1.2)
-    htmlentities (4.3.3)
+    htmlentities (4.3.4)
```

Comment:
These are just loosely pinned to the minor version, usually acceptable to update